command_lib/base.yml

```diff
@@ -87,3 +87,34 @@ debian:
           delimiter: 'EOF'
         src_urls: {}
 
+# openjdk -------------------------------------------------------------------------------------------------------------------------------
+openjdk:
+  latest: '8'
+  tags:
+    '8':
+        shell: '/bin/bash'
+        # either enter the package names as a list or a snippet to invoke either within or outside the container
+        names:
+          invoke:
+            1:
+              # Currently two types of environments are supported - host or container
+              container:
+                - "dpkg --get-selections | cut -f1 -d':' | awk '{print $1}'"
+          delimiter: "\n"
+        # the length of this list should be the same as the names or else there will be a mismatch
+        versions:
+          invoke:
+            1:
+              container:
+                - "pkgs=`dpkg --get-selections | cut -f1 -d':' | awk '{print $1}'`"
+                - "for p in $pkgs; do dpkg -l $p | awk 'NR>5 {print $3}'; done"
+          delimiter: "\n"
+        licenses:
+          invoke:
+            1:
+              container:
+                - "pkgs=`dpkg --get-selections | cut -f1 -d':' | awk '{print $1}'`"
+                - "for p in $pkgs; do cat /usr/share/doc/$p/copyright; echo EOF; done"
+          delimiter: 'EOF'
+        src_urls: {}
```

Comment:
Are you sure you want to leave this empty? Your report will not contain any project urls.

Reply:
Ah, I see.  I used debian as a template.  Let me work on the debian command for getting the project urls.

Reply:
Much appreciated! Thank you!

Reply:
Here is the output I can get about the package.  The APT-Sources line is what we want to capture, correct?

```
Package: wget
Version: 1.18-5+deb9u1
Priority: important
Section: web
Maintainer: Noël Köthe <noel@debian.org>
Installed-Size: 2813 kB
Depends: libc6 (>= 2.17), libgnutls30 (>= 3.5.6), libidn11 (>= 1.13), libnettle6, libpcre3, libpsl5 (>= 0.13.0), libuuid1 (>= 2.16), zlib1g (>= 1:1.1.4)
Recommends: ca-certificates
Conflicts: wget-ssl
Homepage: https://www.gnu.org/software/wget/
Tag: implemented-in::c, interface::commandline, network::client,
 protocol::ftp, protocol::http, protocol::ssl, role::program,
 suite::gnu, use::downloading, works-with::file
Download-Size: 800 kB
APT-Manual-Installed: yes
APT-Sources: http://deb.debian.org/debian stretch/main amd64 Packages
Description: retrieves files from the web
```

Reply:
Yes! That looks correct.